pkgs/development/libraries/libspatialaudio/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, stdenv, fetchFromGitHub, cmake, libmysofa, zlib }:
+stdenv.mkDerivation {
+  name = "libspatialaudio";
+  src = fetchFromGitHub {
+    owner = "videolabs";
+    repo = "libspatialaudio";
+    rev = "0.3.0";
+    hash = "sha256-sPnQPD41AceXM4uGqWXMYhuQv0TUkA6TZP8ChxUFIoI=";
+  };
+  buildInputs = [ cmake libmysofa zlib ];
+  configurePhase = "cmake -DCMAKE_INSTALL_PREFIX=$out .";
```

Comment:
The `cmake` package contains a setup hook that should do something like this automatically.

Reply:
Indeed it does, not sure what I was thinking putting `cmake .` as `configurePhase` anyway. Removing this line makes it all good.